deep_quoridor/src/agents/flat_dqn.py

```diff
@@ -183,7 +183,7 @@ class Pretrained01FlatDQNAgent(FlatDQNAgent):
 
     def __init__(self, board_size, **kwargs):
         super(Pretrained01FlatDQNAgent, self).__init__(board_size)
```

Comment:
```suggestion
        super().__init__(board_size, epsilon=0.0)
```
As I mentioned in chat, epsilon should be 0 when playing, otherwise if you don't specify it, it will be 1 so it will be playing randomly.  Additional nit, I removed the parameters to `super`, that was the format in Python 2.0